### PR TITLE
Remove cringe

### PR DESCRIPTION
**Pull request by chexo3**
I'm not even giving this the respect of cloning it locally, let alone testing it. I deleted the one usage I found of `Wisdom.spread` and deleted `Wisdom.java`.

### CODE REVIEW

**Review by CoolMineman — APPROVED** [review]
:pray:

**Comment by chexo3** [comment]
Sin 1: Cringe
Sin 2: (At least on 1.21.1) "Wisdom" logger/log message doesn't identify itself as part of owo-lib
Sin 3: No way to disable it/on by default

File: src/main/java/io/wispforest/owo/Owo.java [hunk 1]
```diff
@@ -83,4 +81,4 @@ public static MinecraftServer currentServer() {
         return SERVER;
     }
 
-}
\ No newline at end of file
+}
```
Comment (sylv256) [inline src/main/java/io/wispforest/owo/Owo.java:83]: small nitpick, does this repository use this style?

**Review by sylv256 — APPROVED** [review]
LGTM

File: src/main/java/io/wispforest/owo/Owo.java (hunk 1 above)
Comment (chexo3) [reply src/main/java/io/wispforest/owo/Owo.java:83]: I neither know nor care

**Comment by chyzman** [comment]
no <3

**Comment by chexo3** [comment]
Sin 4: Copied an error message/made a "wisdom" that looks like an error message from another mod

`This custom packet isn't supported by Gadget - Add support via DrawPacketHandler.EVENT`

**Comment by chexo3** [comment]
> no <3

have you considered

it's annoying as fuck

**Comment by chyzman** [comment]
> > no <3
> 
> have you considered
> 
> it's annoying as fuck

 precisely why i said it

**Comment by chyzman** [comment]
if you're gonna be a dick with ur PRs, then im gonna be a dick in my replies

**Comment by Dragon-Seeker** [comment]
@chexo3 Well, this will not be accepted without respect to locally cloning the repo and testing the PR. This is also a possible API that people may use so breaking change while on a released version is not best practice.

**Comment by chexo3** [comment]
> > > no <3
> > 
> > 
> > have you considered
> > it's annoying as fuck
> 
> precisely why i said it

I meant the "feature", not your message

though that was annoying too

**Comment by chexo3** [comment]
> @chexo3 Well, this will not be accepted without respect to locally cloning the repo and testing the PR. This is also a possible API that people may use so breaking change while on a released version is not best practice.

1. I'll test it locally after I post this comment.

2. Is it exposed as a public API? This looks like an internal function only used in one place. If it's not exposed for use by other mods then nobody should be using it to begin with and it's their fault if they rely on it.

**Comment by chyzman** [comment]
it is definitely used by at least 1 mod

**Comment by chexo3** [comment]
> it is definitely used by at least 1 mod

"If it's not exposed for use by other mods then nobody should be using it to begin with and it's their fault if they rely on it."

Also, it's still cringe, an anti-feature, and arguably user-hostile.

**Comment by chyzman** [comment]
> > it is definitely used by at least 1 mod
> 
> "If it's not exposed for use by other mods then nobody should be using it to begin with and it's their fault if they rely on it."
> 
> Also, it's still cringe, an anti-feature, and arguably user-hostile.

honestly, if you're going to be upset by a silly inside joke when you start your game, you deserve to be "hurt"

**Comment by chexo3** [comment]
> upset by a silly inside joke when you start your game

I don't care about the "joke" it's annoying, at least one of them can be mistaken for an actual error message, and the whole thing is just generally unfunny.

**Comment by sylv256** [comment]
This isn't a very professional response from a mod development team.

**Comment by chexo3** [comment]
> > > it is definitely used by at least 1 mod
> > 
> > 
> > "If it's not exposed for use by other mods then nobody should be using it to begin with and it's their fault if they rely on it."
> > Also, it's still cringe, an anti-feature, and arguably user-hostile.
> 
> honestly, if you're going to be upset by a silly inside joke when you start your game, you deserve to be "hurt"

also what year is it? is this not basically the "hurr durr snowflake" response? 4chan called, they want their rhetorical device back

**Comment by chexo3** [comment]
@Dragon-Seeker Tested and working. I forgot to remove the import statement for the deleted class, build failed as a result. I fixed it, tested the build artifact, and it appears to work. I guess I can also run the unit tests, I didn't check if those exist yet.

**Comment by chexo3** [comment]
I attempted to run the tests, but it won't build on my machine. Odd. Well, Github Actions' artifact still worked fine.

**Comment by Dragon-Seeker** [comment]
> Also, it's still cringe, an anti-feature, and arguably user-hostile.

This will be my last response to this post as I would have closed this issue but sadly must wait for @gliscowo to look at such. 

Taken from [Wiktionary](https://en.wiktionary.org/wiki/anti-feature), an anti-feature is a "Intentionally-implemented [functionality](https://en.wiktionary.org/wiki/functionality#English) of a product or service (typically technology) which hinders or disadvantages the [user](https://en.wiktionary.org/wiki/user#English), and which the seller may charge users to not include." This means that a single log statement at the start of a log is hindering enough to lead to issues leading to disadvantages when diagnosing issues. I do think the message that looks like an error is something that possible should be removed as it could be seen as misleading but the concept itself does little to the already problematic situation of log clutter.

Also Taken from [Wiktionary](https://en.wiktionary.org/wiki/user-hostile), user-hostile means "[Difficult](https://en.wiktionary.org/wiki/difficult#English) to use, especially for an [untrained](https://en.wiktionary.org/wiki/untrained#English) user." meaning that this prioritizes the needs of the Dev on a feature rather than ones who use such which I would say many users do not look within the logs but on the off chance, many would look past this for the real error. If such thought this was an error then I would be more confused than that one message that could trip someone up.

I think its existence is based on an old premise where mods talked to each other within the logs as a cute laugh and many of the messages are only there as funny jokes. Frankly, I find it funny seeing such messages and overall I believe this is fueled by more than a issue with one message, and doubtful you will get the outcome desired by this PR.

**Comment by Dragon-Seeker** [comment]
> This isn't a very professional response from a mod development team. 

Chyzman is a contributor of the team but his actions most of the time are of his own and more or less can be a bit harsh the problem is that this keeps coming up and frankly can understand that he leaves no care to this situation besides giving his cynical answer.

**Review by sylv256 — APPROVED** [review]
LGTM 2

**Pull request closed without merging**